web/styles/right_sidebar.css

```diff
@@ -13,11 +13,30 @@ $user_status_emoji_width: 24px;
     overflow: auto;
 }
 
-#user_presences {
-    max-width: 95%;
+.toggle-narrow-users,
+.toggle-other-users {
+    width: 7px;
+    margin-top: 4px;
```

Comment:
Looks like you can center this vertically by just setting `align-items: center` on the flex container and ditching the `margin-top` here. You might have to fiddle some with the height or line-height of the icon, but in testing just now, it looks pretty good to me.

Reply:
Thanks for noticing that, updated to do that instead!